src/sdk-reference/bulk/import/import.js.md

```diff
@@ -0,0 +1,50 @@
+# Import
+
+```javascript
+import(data, options)
```

Comment:
Can you add the return value and optional arguments ?
(Eg: `create(index, options = {}) ⇒ object`)